main.go

```diff
@@ -20,6 +21,12 @@ var log = loggo.GetLogger("main")
 var branch string
 var revision string
 
+var CONFIG_FILE_LOCATIONS = []string{
+	"/etc",
+	"/var/secrets",
```

Comment:
This was `/run/secrets` before (for some reason) and I'd like to keep it that way to preserve backwards compatibility. Otherwise looks good!

Reply:
Whoops, good catch